### PR TITLE
[Streaming] Test build fixed

### PR DESCRIPTION
**Pull request by lixin-wei**
## Why are these changes needed?

Fix test building in Streaming CI.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java [hunk 1]
```diff
@@ -8,13 +9,18 @@
...
   @Test
   public void testCreation() {
-    JobMaster jobMaster = new JobMaster(new HashMap<>());
-    Assert.assertNotNull(jobMaster.getRuntimeContext());
-    Assert.assertNotNull(jobMaster.getConf());
-    Assert.assertNull(jobMaster.getGraphManager());
-    Assert.assertNull(jobMaster.getResourceManager());
-    Assert.assertNull(jobMaster.getJobMasterActor());
-    Assert.assertFalse(jobMaster.init(false));
+    try {
```
Comment (chaokunyang) [inline streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java:12]: extend BaseTest instead

Comment (lixin-wei) [reply streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java]: done.

**Review by chaokunyang — APPROVED** [review]
LGTM

File: streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
 import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-public class JobMasterTest {
+public class JobMasterTest extends BaseUnitTest {
+
+  @BeforeMethod
+  public void init() {
```
Comment (chaokunyang) [inline streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java:14]: I think parent class has those methods

**Review by chaokunyang — DISMISSED** [review]

File: streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java (hunk 2 above)
Comment (lixin-wei) [reply streaming/java/streaming-runtime/src/test/java/io/ray/streaming/runtime/master/JobMasterTest.java:14]: No, it doesn't.

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**